@xen-orchestra/lite/src/views/pool/PoolTasksView.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-  <UiCard>
+  <UiCard :color="hasError ? 'error' : undefined">
     <UiTitle class="title-with-counter" type="h4">
       {{ $t("tasks") }}
       <UiCounter :value="pendingTasks.length" color="info" />
     </UiTitle>
 
     <TasksTable :finished-tasks="finishedTasks" :pending-tasks="pendingTasks" />
-    <UiSpinner v-if="!isReady" class="loader" />
+    <UiSpinner v-if="!isReady" />
```

Comment:
Is it intentional to display `TasksTable` even if `isReady === false`?

Reply:
Since the loader in now handled inside TasksTable, I think this `UiSpinner` can be removed.